client/src/constants/regExp.ts

```diff
@@ -3,6 +3,7 @@ const REGEXP = {
   eventUrl: /\/event\/([a-zA-Z0-9-]+)\//,
   billTitle: /^([ㄱ-ㅎ가-힣a-zA-Z0-9ㆍᆢ]\s?)*$/,
   memberName: /^([ㄱ-ㅎ가-힣a-zA-Zㆍᆢ]\s?)*$/,
+  accountNumber: /^[0-9\s\-]*$/,
```

Comment:
```suggestion
  accountNumber: ^\d+([\s\-]\d+)*[\s\-]?$,
```
로 사용하면 연속된 -나 공백을 걸러줄 수 있어요~

Reply:
토다리는 정(규표현식의)신